Components/Core/Inc/Command.hpp

```diff
@@ -64,21 +67,21 @@ class Command
     void SetTaskCommand(uint16_t taskCommand) { this->taskCommand = taskCommand; }
     void SetDataSize(uint16_t size) { dataSize = size; }
 
-
-protected:
+   protected:
     // Data -- note each insertion and removal from a queue will do a full copy of this object, so this data should be as small as possible
-    GLOBAL_COMMANDS command;    // General GLOBAL command, each task must be able to handle these types of commands
-    uint16_t taskCommand;        // Task specific command, the task this command event is sent to needs to handle this
+    GLOBAL_COMMANDS command;  // General GLOBAL command, each task must be able to handle these types of commands
+    uint16_t taskCommand;     // Task specific command, the task this command event is sent to needs to handle this
 
-    uint8_t* data;                // Pointer to optional data
-    uint16_t dataSize;            // Size of optional data
+    uint8_t* data;      // Pointer to optional data
+    uint16_t dataSize;  // Size of optional data
 
-private:
-    bool bShouldFreeData;        // Should the Command handle freeing the data pointer (necessary to enable Command object to handle static memory ptrs)
+   private:
+    bool
```

Comment:
weird that this is two lines I feel like the commit should be on another line instead of the var name

Reply:
Due to column limit. Unfortunately no way to change what gets moved to next line.

https://github.com/StudentOrganisationForAerospaceResearch/AvionicsSoftware/pull/86#discussion_r1402846522